CHANGELOG.md

```diff
@@ -5,6 +5,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.6.0] - 2022-02-11
+### Removed
+- Removes `signalCB` param (is replaced with `signal` event)
+
+### Added
+
+- Added option for `anonymous_allowed` among the customized authentication options
+
+- Adds `unrecoverableAgentState` and `signal` to alert types
```

Comment:
bikeshedding: I think we discussed `kebab-case` for event names. so `unrecoverable-agent-state` instead